defaults/main.yml

```diff
@@ -9,14 +9,21 @@ sonar_version_directory: "sonarqube-{{ sonar_version }}"
 
 sonar_web_context: ''
 
-# MySQL database connection details.
-sonar_mysql_username: sonar
-sonar_mysql_password: sonar
+# The Database engine.
+# possible values: postgres and mysql
+sonar_database_engine: postgres
```

Comment:
I think that the default database engine should be mysql to avoid regression.

Reply:
thanks for your recommendation. mysql wasn't working that's why I've changed the default to postgres.